acsoo/addons.py

```diff
@@ -85,3 +92,100 @@ def addons_list_depends(ctx, exclude):
 
 
 addons.add_command(addons_list_depends, 'list-depends')
+
+
+@click.command(help="Print a comma separated list of the modified installable "
+                    "addons from the given git ref")
+@click.argument('git_ref')
+@click.option('-r', '--diff-requirements', 'diff_requirements', is_flag=True,
+              help="Defines whether the comparison must take the requirements "
+                   "file into account or not.")
+@click.option('--exclude', default='',
+              help="Comma separated list of addons to exclude from "
+                   "the dependencies.")
+@click.pass_context
+def addons_toupdate(ctx, git_ref, diff_requirements, exclude):
```

Comment:
Please extract this method in a new module (`addons_toupdate.py`) so all addons subcommands don't end up in the same file. cc/ @adrienpeiffer